tasks/WhereIsThis.tex

```diff
@@ -32,28 +31,18 @@ \subsection*{Additional rules and remarks}
 		\item \textbf{Follower's assistance:} Helping the robot while guiding a guest (e.g.~waving, shouting) causes a score reduction of 50pts.
 	\end{itemize}
 
-	\item \textbf{Information Point:} The robot meets guests and gives directions at the \emph{Information Point}.
-	After guiding a person to its destination, it must return to the information point.
+	\item \textbf{Information Point:} The robot meets guests and gives directions at the \emph{Information Point} and then guides them to the location.
+	After guiding a person, it must return to the information point.
```

Comment:
We should add that the information point is not known and random for every team. Maybe:

The information point is not known beforehand. Right before a run at the entrance door, the team is told a position where they should start the robot.

Reply:
Agreed. Added.